### PR TITLE
PHPStan level 3

### PR DESCRIPTION
**Pull request by yguedidi**

### CODE REVIEW

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+        -
+            message: "#^Method Wallabag\\\\AnnotationBundle\\\\Controller\\\\WallabagAnnotationController\\:\\:postAnnotationAction\\(\\) should return Symfony\\\\Component\\\\HttpFoundation\\\\JsonResponse but returns Symfony\\\\Component\\\\Form\\\\FormInterface\\<mixed\\>\\.$#"
+            count: 1
+            path: src/Wallabag/AnnotationBundle/Controller/WallabagAnnotationController.php
+
+        -
+            message: "#^Method Wallabag\\\\AnnotationBundle\\\\Controller\\\\WallabagAnnotationController\\:\\:putAnnotationAction\\(\\) should return Symfony\\\\Component\\\\HttpFoundation\\\\JsonResponse but returns Symfony\\\\Component\\\\Form\\\\FormInterface\\<mixed\\>\\.$#"
+            count: 1
+            path: src/Wallabag/AnnotationBundle/Controller/WallabagAnnotationController.php
+
```
Comment (yguedidi) [inline phpstan-baseline.neon:12]: to me returning the form looks wrong here, what is expected?

File: phpstan.neon [hunk 2]
```diff
@@ -14,3 +14,6 @@ parameters:
         - vendor/bin/.phpunit/phpunit-8.5-0/vendor/autoload.php
 
     inferPrivatePropertyTypeFromConstructor: true
+
+    ignoreErrors:
+        - "#^Method Symfony\\\\Contracts\\\\EventDispatcher\\\\EventDispatcherInterface\\:\\:dispatch\\(\\) invoked with 2 parameters, 1 required\\.$#"
```
Comment (yguedidi) [inline phpstan.neon:19]: will be removed by the upgrade to Symfony 5

File: phpstan-baseline.neon (hunk 1 above)
Comment (yguedidi) [reply phpstan-baseline.neon:12]: anyway it's only 2 errors in the baseline, PR can be accepted as is for now IMO

**Review by nicosomb — APPROVED** [review]

**Review by nicosomb — APPROVED** [review]

**Review by j0k3r — APPROVED** [review]

**Comment by yguedidi** [comment]
@j0k3r thanks! I see this is in the 2.6.4 milestone, why? shouldn't we limit only bugfixes in 2.6.x from now on and have any new feature or anything else that is not a bugfix be in master (the future 2.7.x)?
except if increasing type coverage is considered a bug fix 😅

**Comment by j0k3r** [comment]
Oups yes sorry

**Pull request merged**